matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/livelocation/LiveLocationAggregatedSummary.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.session.room.model.livelocation
+
+import org.matrix.android.sdk.api.session.room.model.message.MessageLiveLocationContent
+
+/**
+ * Aggregation info concerning a live location share.
+ */
+data class LiveLocationAggregatedSummary(
+        /**
+         * Event id of the event that started the live.
+         */
+        val eventId: String,
+        val roomId: String,
+        val isLive: Boolean?,
```

Comment:
is it possible to have a live location that isn't live?

Reply:
Yes, in the timeline we want to see the lives that have been stopped or timed out. Do you think a comment on the field would help?

Reply:
Not to fuel the naming discussion but worth noting that `isActive` can be `true` even when live location sharing is _not_ currently active because the current status is the combination of this field and `endOfLiveTimestampAsMilliseconds`. The previous `isLive` simply followed the naming in the event body which may (or may not) have made this less confusing.

Reply:
Good point. The SDK is in the middle of the Matrix spec, and the client application. It's not always easy to please both side. For instance an `Event` has a field `sender`, but the SDK [renamed it](https://github.com/vector-im/element-android/blob/main/matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt#L85) to `senderId`, which is more precise.

In this case, `LiveLocationAggregatedSummary` is a type which is build by the SDK (so a bit far from the spec) and exposed to the App. So the fields name should follow the app needs. It's also possible (and even recommended) to document the fields using `/** doc */` to avoid any ambiguity.

Naming is one of the most difficult parts of our job :)

Reply:
Ok, sounds good. 🙂